Automattic/WP-Job-Manager
# Add download field to writepanels

This PR is tied to https://github.com/Automattic/wp-job-manager-resumes/pull/319, but works without broke if only this is merged.

### Changes proposed in this Pull Request

* Introduces support to a custom download field in the writepanels.

### Testing instructions

* Checkout this along this branch: https://github.com/Automattic/wp-job-manager-resumes/pull/319.
* Go to the Resumes settings and activate the option `Resume Upload`.
* Go to the `Submit Resume` page and submit a resume with a resume file.
* Go to the WP-admin > Resumes.
* Open the created resume.
* In the `Resume File` click in view and make sure it's downloaded through a custom URL with the query string: `?download-resume={XX}&file-id=0`. Where the `{XX}` is the post ID.
